src/sortStudents.ts

```diff
@@ -1,7 +1,78 @@
-// describe Student type
-// create and export SortType enum
-// create SortOrder type
+type Student = {
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+};
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+export enum SortType {
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+enum SortOrder {
+  asc = 'asc',
+  desc = 'desc',
```

Comment:
```suggestion
  Ascending = 'asc',
  Descending = 'desc',
```
Use more meaningful names